oaknational/Oak-Web-Application
# Feat/downloads confirmation state LESQ-408

## Description

- Creates download confirmation component and test file
- Adds UI graphics

## Issue(s)

Fixes #LESQ-410

## Acceptance Criteria:

- [x]  When a download is successfully completed show the thank you message
- [x]  `Back to lesson` link to the lesson that the download was for
- [x]  Show the tick mark happiness icon
- [x]  Do all of this on the same lesson download page (a back in browser on this page will take the user to the lesson page)
- [x]  ‘More lesson’ section is a separate ticket [[LESQ-306](https://www.notion.so/I-want-a-download-success-experience-that-makes-it-easy-to-find-and-download-more-lessons-in-the-sam-7fcd6da2856b4309a3cdbca3093e215d?pvs=21)](https://www.notion.so/I-want-a-download-success-experience-that-makes-it-easy-to-find-and-download-more-lessons-in-the-sam-7fcd6da2856b4309a3cdbca3093e215d?pvs=21)

## How to test

1. Go to https://deploy-preview-2068--oak-web-application.netlify.thenational.academy/teachers/programmes/combined-science-secondary-ks4-foundation-aqa/units/measuring-waves/lessons/transverse-waves/downloads?preselected=all
2. Download a resource 
3. You should see download confirmation component appears 

## Screenshots

How it used to look (delete if n/a):
<img width="1219" alt="Screenshot 2023-11-02 at 08 07 16" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/0a58b6f3-d8b2-40cd-883b-04d4cbb747ac">

How it should now look:
<img width="1210" alt="Screenshot 2023-11-02 at 07 58 10" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/9ba4b129-80a2-41f8-b357-294c1058e27b">

## Checklist
